spec/features/topics_spec.rb

```diff
@@ -23,6 +23,14 @@
     click_link("#{topic.title}")
     expect(page).to have_css("a", text: "#{bookmark.url}")
   end
+  scenario "User can create new topic" do 
+    visit topics_path
+    click_link("New Topic")
+    fill_in 'Title', with: 'animals'
+    click_button 'Create Topic'
+    expect(page).to render(:topic)
```

Comment:
That was a mistake. I was trying to test the page redirected to the
topics_path. I changed it to :
expect(current_path).to eq(topics_path)

On Fri, Feb 27, 2015 at 8:39 AM, Eliot Sykes <notifications@github.com>
wrote:

> In spec/features/topics_spec.rb
> <https://github.com/LuisMenjivar/luisbookmark/pull/6#discussion_r25506244>
> :
>
> > @@ -23,6 +23,14 @@
> >      click_link("#{topic.title}")
> >      expect(page).to have_css("a", text: "#{bookmark.url}")
> >    end
> > +  scenario "User can create new topic" do
> > +    visit topics_path
> > +    click_link("New Topic")
> > +    fill_in 'Title', with: 'animals'
> > +    click_button 'Create Topic'
> > +    expect(page).to render(:topic)
>
> What's this line testing?
>
> expect(page).to render(:topic)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/luisbookmark/pull/6/files#r25506244>.
>